it/content/src/main/content/jcr_root/content/forms/af/core-components-it/samples/tabsontop/basic/.content.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:fd="http://www.adobe.com/aemfd/fd/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="cq:Page">
+    <jcr:content
+        cq:deviceGroups="[mobile/groups/responsive]"
+        cq:lastModified="{Date}2021-12-02T15:12:59.900+05:30"
+        cq:lastModifiedBy="admin"
+        cq:template="/conf/core-components-it/settings/wcm/templates/af-blank-v2"
+        jcr:language="en"
+        jcr:primaryType="cq:PageContent"
+        jcr:title="Adaptive Form V2 (IT)"
+        sling:resourceType="forms-components-examples/components/page">
+        <guideContainer
+            fd:version="2.0"
+            fieldType="form"
+            jcr:primaryType="nt:unstructured"
+            sling:resourceType="forms-components-examples/components/form/container">
+            <tabsontop
+                    jcr:primaryType="nt:unstructured"
+                    jcr:title="Panel"
+                    sling:resourceType="forms-components-examples/components/form/tabsontop"
+                    assistPriority="name"
+                    description="&lt;p>panel long&lt;/p>&#xd;&#xa;"
+                    enabled="{Boolean}true"
+                    fieldType="panel"
+                    maxItems="4"
+                    minItems="0"
+                    name="tabsontop1662449848339"
+                    roleAttribute="toolbar"
```

Comment:
i think you don't have a role attribute in tabs

Reply:
removed